src/components/DeckMechanisms/index.js

```diff
@@ -343,6 +385,22 @@ export default class DeckMechanisms extends React.Component {
         break
       }
 
+      // Spellbinder Zhevana
+      case 'W8': {
+        if (this.props.mode === 'AUTOMATIC') {
```

Comment:
I don’t think we made that check for the other ones, so we probably can remove it.

Reply:
The others actually don't add mana, they only modify the internal variable frozenEnemiesLevel. In MANUAL mode, the best would be to ask the user to input how many units got destroyed

Reply:
Right, that makes sense, but at this stage, the manual mode doesn’t provide this option so maybe we don’t check the mode at all right now.

Reply:
This means that Zhevana will give random amounts of mana in the deck tracker, but it's probably not that big of an issue since right now her ability is not implemented at all

Reply:
I've removed the check